lmfit/minimizer.py

```diff
@@ -2066,7 +2066,7 @@ def ampgo(self, params=None, max_nfev=None, **kws):
         except AbortFitException:
             pass
 
-        if not result.aborted:
+        if 'ret' in locals():
```

Comment:
please have AbortFitException set `ret = None`
and then make the test here be `if ret is not None`.